vue_app/src/assets/styles/import.scss

```diff
@@ -0,0 +1,2 @@
+@import url('./_variables.scss');

```

Comment:
you can change these to direct import statements! [Your CSS pipeline will take care of the rest I think](https://sass-lang.com/documentation/at-rules/import)